Dockerfile

```
@@ -0,0 +1,13 @@
+FROM rust:1.57-slim-bullseye as build
+WORKDIR /app
+COPY Cargo.* .
+RUN mkdir src && \
+    touch src/lib.rs && \
+    cargo build --release
+COPY src src
+RUN cargo build --release
+
+FROM gcr.io/distroless/cc
+ENV HOST "0.0.0.0"
+COPY --from=build /app/target/release/api_actix-web_rust_hello-world /app/
+CMD ["/app/api_actix-web_rust_hello-world"]
```

I think we could add some information about the exposed port. We're also missing to change the user so we don't run the app as `root` 🙂 
```suggestion
FROM gcr.io/distroless/cc
USER 1000
EXPOSE 6060
ENV HOST "0.0.0.0"
COPY --from=build /app/target/release/api_actix-web_rust_hello-world /app/
CMD ["/app/api_actix-web_rust_hello-world"]
```